From c8b965be5b61c2412df938e1feb380b69bd92ee9 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Thu, 06 Apr 2023 07:15:03 +0000
Subject: [PATCH] [unbundle] do not use ZipAnalyzer traits header

It is enough to include chrome/common/safe_browsing/archive_analyzer_results.h.

Bug: None
Change-Id: Ia63a66b77c073fa4fdb8470db34b4c24722293cc
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/4373706
Reviewed-by: Daniel Rubery <drubery@chromium.org>
Commit-Queue: Stephan Hartmann <stha09@googlemail.com>
Cr-Commit-Position: refs/heads/main@{#1127041}
---

diff --git a/chrome/services/file_util/public/mojom/BUILD.gn b/chrome/services/file_util/public/mojom/BUILD.gn
index b9e80cc..461d2d0 100644
--- a/chrome/services/file_util/public/mojom/BUILD.gn
+++ b/chrome/services/file_util/public/mojom/BUILD.gn
@@ -55,7 +55,8 @@
           cpp = "::safe_browsing::ArchiveAnalyzerResults"
         },
       ]
-      traits_headers = [ "//chrome/common/safe_browsing/zip_analyzer.h" ]
+      traits_headers =
+          [ "//chrome/common/safe_browsing/archive_analyzer_results.h" ]
       traits_private_headers = [ "safe_archive_analyzer_param_traits.h" ]
       traits_public_deps = [
         "//chrome/common/safe_browsing:proto",
diff --git a/chrome/services/file_util/safe_archive_analyzer.h b/chrome/services/file_util/safe_archive_analyzer.h
index 62307b8..0231205 100644
--- a/chrome/services/file_util/safe_archive_analyzer.h
+++ b/chrome/services/file_util/safe_archive_analyzer.h
@@ -7,6 +7,7 @@
 
 #include "chrome/common/safe_browsing/archive_analyzer_results.h"
 #include "chrome/common/safe_browsing/rar_analyzer.h"
+#include "chrome/common/safe_browsing/zip_analyzer.h"
 #include "chrome/services/file_util/public/mojom/safe_archive_analyzer.mojom.h"
 #include "mojo/public/cpp/bindings/remote.h"
 
